mava/configs/env/connector.yaml

```diff
@@ -0,0 +1,9 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: con-10x10x5a
+
+env_name: MaConnector-v2
```

Comment:
Since this is only used for logging can this just be `MaConnector` please? And can you add a comment mentioning this in this file like we have in other env configs as well please?